### PR TITLE
chore(nuxt-deps): update nuxt dependencies & metatypes/units-typpography

### PR DESCRIPTION
**Pull request by carsoli**

### CODE REVIEW

File: packages/documentation/nuxt.config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const config = {
 		link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }],
 	},
 	css: ['./assets/css/main.scss'],
-	modules: ['@nuxtjs/markdownit', '@nuxtjs/axios', '@nuxtjs/style-resources'],
+	modules: ['@nuxtjs/markdownit', '@nuxtjs/style-resources'],
```
Comment (carsoli) [inline packages/documentation/nuxt.config.js:23]: this was never used and looks like a setup blind following artifact

File: packages/documentation/package.json [hunk 2]
```diff
@@ -14,15 +14,14 @@
 	},
 	"description": "Kotti-ui Doc Pages",
 	"devDependencies": {
-		"@metatypes/typography": "^0.4.0",
-		"@metatypes/units": "^0.4.0",
+		"@metatypes/typography": "^0.5.0",
```
Comment (carsoli) [inline packages/documentation/package.json:17]: unrelated to nuxt deps but v simple to

**Review by FlorianWendelborn — APPROVED** [review]

File: packages/documentation/package.json [hunk 3]
```diff
@@ -14,15 +14,14 @@
...
-		"@nuxt/typescript-runtime": "^0.4.10",
-		"@nuxtjs/axios": "^5.11.0",
-		"@nuxtjs/markdownit": "^1.2.7",
-		"@nuxtjs/markdownit-loader": "^1.1.1",
-		"@nuxtjs/style-resources": "^1.0.0",
+		"@nuxt/typescript-build": "2.x",
+		"@nuxt/typescript-runtime": "2.x",
+		"@nuxtjs/markdownit": "2.x",
+		"@nuxtjs/markdownit-loader": "1.x",
+		"@nuxtjs/style-resources": "1.x",
```
Comment (carsoli) [inline packages/documentation/package.json:24]: 2.x is meant for nuxt 3

File: packages/documentation/package.json [hunk 4]
```diff
@@ -14,15 +14,14 @@
...
+		"@metatypes/units": "^0.5.0",
 		"@nuxt/types": "^2.13.2",
-		"@nuxt/typescript-build": "^1.0.3",
-		"@nuxt/typescript-runtime": "^0.4.10",
-		"@nuxtjs/axios": "^5.11.0",
-		"@nuxtjs/markdownit": "^1.2.7",
-		"@nuxtjs/markdownit-loader": "^1.1.1",
-		"@nuxtjs/style-resources": "^1.0.0",
+		"@nuxt/typescript-build": "2.x",
+		"@nuxt/typescript-runtime": "2.x",
```
Comment (carsoli) [inline packages/documentation/package.json:21]: this package becomes obsolete from Nuxt 2.15 onwards but we can't upgrade to nuxt 2.15 without upgrading vue-loader to 2.7 (which is the naruto upgrade branch)

**Pull request merged**